src/functionals/rSCANc.cpp

```diff
@@ -0,0 +1,26 @@
+#include "functional.hpp"
+#include "constants.hpp"
+#include "SCAN_like_eps.hpp"
+
+template<class num>
+static num rSCANC(const densvars<num> &d)
+{
+  num eps_c = SCAN_eps::SCAN_C(d, 1, 1, 0); 
+ 
+  return eps_c;
+}
+
+FUNCTIONAL(XC_RSCANC) = {
+  "rSCAN correlation functional",
+  "rSCAN correlation functional.\n"
+  "Regularised SCAN functional\n"
+  "A. P. Bartók and J. R. Yates, J. Chem. Phys. 150, 161101 (2019)."
```

Comment:
I suggest you substitute "ó" with "o". I think it's better to avoid special characters.

Reply:
The citations came straight from my referencing program, I hadn't thought about the special character, good spot. It's changed.